From 612947e86ac32b7600ec7f28961259089a69281d Mon Sep 17 00:00:00 2001
From: David Boddie <david.boddie@qt.io>
Date: Fri, 6 Jun 2025 17:39:45 +0200
Subject: [PATCH 228/553] Doc: Fix broken "see also" links

Pick-to: 6.9
Change-Id: I9a7386691d4a852bb687310d6c7a91b78328fe8c
Task-number: QTBUG-137048
Reviewed-by: Paul Wicking <paul.wicking@qt.io>
(cherry picked from commit 253dadeb72b1e3caee46919d0b896c24efb10b91)
Reviewed-by: Qt Cherry-pick Bot <cherrypick_bot@qt-project.org>
---
 src/corelib/tools/qhash.cpp      | 6 +++---
 src/corelib/tools/qmultimap.qdoc | 2 +-
 src/corelib/tools/qspan.qdoc     | 2 +-
 3 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/src/corelib/tools/qhash.cpp b/src/corelib/tools/qhash.cpp
index 1cc5b2fb883..a286ff933ae 100644
--- a/src/corelib/tools/qhash.cpp
+++ b/src/corelib/tools/qhash.cpp
@@ -3316,7 +3316,7 @@ size_t qHash(long double key, size_t seed) noexcept
     Returns \c true if the hash contains an item with the \a key and
     \a value; otherwise returns \c false.
 
-    \sa contains()
+    \sa count()
 */
 
 /*!
@@ -3326,7 +3326,7 @@ size_t qHash(long double key, size_t seed) noexcept
     Removes all the items that have the \a key from the hash.
     Returns the number of items removed.
 
-    \sa remove()
+    \sa remove(const Key &key, const T &value)
 */
 
 /*!
@@ -3429,7 +3429,7 @@ size_t qHash(long double key, size_t seed) noexcept
 
     Returns the number of items with the \a key and \a value.
 
-    \sa count()
+    \sa contains()
 */
 
 /*!
diff --git a/src/corelib/tools/qmultimap.qdoc b/src/corelib/tools/qmultimap.qdoc
index 8760bf93bf7..b7d5a019b72 100644
--- a/src/corelib/tools/qmultimap.qdoc
+++ b/src/corelib/tools/qmultimap.qdoc
@@ -732,7 +732,7 @@
     If the multi map contains no item with key \a key, the function
     returns constEnd().
 
-    \sa find(), QMultiMap::constFind()
+    \sa find(), QMultiMap::constFind(const Key &key, const T &value)
 */
 
 /*!
diff --git a/src/corelib/tools/qspan.qdoc b/src/corelib/tools/qspan.qdoc
index 93f4b83c3a9..6acdc8ac0e8 100644
--- a/src/corelib/tools/qspan.qdoc
+++ b/src/corelib/tools/qspan.qdoc
@@ -582,7 +582,7 @@
     \c{Offset + Count} \e{and} size() >= \c{Offset + Count}), otherwise the
     behavior is undefined.
 
-    \sa subspan(QSpan<T,E>::size_type, QSpan<T,E>::size_type), subspan(), first(), last()
+    \sa subspan(QSpan<T,E>::size_type, QSpan<T,E>::size_type), first(), last()
     \sa {Variable-Size and Fixed-Size Spans}
 */
 
-- 
2.51.2

